Echo.Core/src/Textures/Colors/RGBA128.Parser.cs

```diff
@@ -141,9 +145,84 @@ bool ParseHex(out RGBA128 result)
 			static int CombineHex(int hex0, int hex1) => hex0 * 16 + hex1;
 		}
 
-		bool ParseRGB(out RGBA128 result) => throw new NotImplementedException();
+		bool ParseRGB(out RGBA128 result)
+		{
+			int index = 0;
+
+			int r = 0, g = 0, b = 0, a = 255;
+
+			if (findNextInt(content, ref index, ref r) &&
+				findNextInt(content, ref index, ref g) &&
+				findNextInt(content, ref index, ref b))
+			{
+				if (findNextInt(content, ref index, ref a)) return YieldError(out result);
+
+
+				if (IsChannelInRange(r) &&
+					IsChannelInRange(g) &&
+					IsChannelInRange(b) &&
+					IsChannelInRange(a))
+				{
+					result = new RGBA128(r / 255f, g / 255f, b / 255f, a / 255f);
+					return true;
+				}
+
+			}
+			return YieldError(out result);

```

Comment:
There is a method down below called `ConvertRGBA` that performs the same function.